src/test/shell/bazel/bazel_coverage_cc_test_llvm.sh

```diff
@@ -150,17 +155,22 @@ end_of_record"
 }
 
 function test_cc_test_with_runtime_objects_not_in_runfiles() {
-  local -r llvm_profdata="/usr/bin/llvm-profdata-9"
-  if [[ ! -x ${llvm_profdata} ]]; then
+  local -r clang="/usr/bin/clang"
+  if [[ ! -x ${clang} ]]; then
     return
   fi
+  local -r clang_version=$(clang --version | grep -o "clang version [0-9]*" | cut -d " " -f 3)
+  if [ "$clang_version" -lt 9 ] || [ "$clang_version" -eq 10 ] || [ "$clang_version" -eq 11 ]; then
+    # No lcov produced with <9.0, no branch coverage with 10.0 and 11.0.
+    echo "clang versions <9.0 as well as 10.0 and 11.0 are not supported." && return
```

Comment:
So versions 9 and 12+ are good? That's a really broad range we don't test on...

@meteorcloudy can we be certain that at least some of our CI pipelines pass these constraints?

Reply:
I'm not sure, but you can login to the docker image and check the clang version.

You can find out which platforms we test Bazel on here: https://github.com/bazelbuild/bazel/blob/master/.bazelci/postsubmit.yml#L3

And the corresponding docker image for a platform here:
https://github.com/bazelbuild/continuous-integration/blob/master/buildkite/bazelci.py#L488

Reply:
And I guess @fmeum discovered this because there were actually test failures in presubmit?

Reply:
@meteorcloudy Well, not really: This test started out as requiring 9. Since I don't have such an old version available locally, the tests ended up being skipped there, so I removed the pin. Then testes started to fail in presubmit, likely because some machine uses 10 or 11. That's how I came up with the current check - I know that it is run on at least *some* machines this way.

Reply:
If it's run on at least one machine with a version higher than 11.0, then that's enough for me.

It's not ideal, but I don't see that ever being downgraded.